Library/Homebrew/extend/os/linux/diagnostic.rb

```diff
@@ -59,6 +61,36 @@ def check_umask_not_zero
             echo 'umask 002' >> #{shell_profile}
         EOS
       end
+
+      def check_glibc_minimum_version
+        return unless OS::Linux::Glibc.below_minimum_version?
+
+        <<~EOS
+          Your glibc is too old (#{OS::Linux::Glibc.system_version}).
+          We only support glibc #{OS::Linux::Glibc.minimum_version} or later.
+          You will encounter build failures and other breakages. Please
```

Comment:
```suggestion
          You may encounter build failures and other breakages. Please
 ```
Build failures are not expected, since it'll use brewed glibc. We're just being conservative.

Reply:
Will do